### PR TITLE
Rename main.css to style.css in build process

### PR DESCRIPTION
**Pull request by sthiepaan**
## General
**Reference:** #2339 
**Description:**
- Change _CSS_ output file name to `style.css`
- Fix test against _CSS_ file
- Update documentation after output file rename
- Bump `package.json` after running `npm run build`
- Documents change in `CHANGELOG.md`

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Additional notes
As I noticed, the styling for boilerplate come from the _NPM Package_ called _main.css_ and it seems to be a bit confusing on a first look. Since that package is strongly connected to this project, we might consider:
- rename it to something that will not affect filename e.g.: _html5-boilerplate-styling_
- reorganize repositories to use _scoped packages_ (with `@` symbol) e.g.:  _@html5-boilerplate/styling_

Pull requests should be thought of as a conversation. There will be some back and forth when trying to get code merged into this or any other project. With all but the simplest changes you can and should expect that the maintainers of the project will request changes to your code. Please be aware of that and check in after you open your PR in order to get your code merged in cleanly.

Thanks!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 9.0.0 (TBD)
```
Comment (roblarsen) [inline CHANGELOG.md:3]: Remove this. I have an open PR with a line for these changes, I'll link to your PR when it's merged in.

File: dist/doc/css.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Several base styles are included that build upon `Normalize.css`. These styles:
 * and more...
 
 These styles are included in
-[main.css](https://github.com/h5bp/html5-boilerplate/blob/master/dist/css/main.css).
-See the [main.css](https://github.com/h5bp/main.css) project
-[documentation](https://github.com/h5bp/main.css/blob/master/README.md#features)
+[style.css](https://github.com/h5bp/html5-boilerplate/blob/master/dist/css/style.css)
+using [main.css](https://github.com/h5bp/main.css) project.
+See the [documentation](https://github.com/h5bp/main.css/blob/master/README.md#features)
```
Comment (roblarsen) [inline dist/doc/css.md:43]:
```suggestion
See the main.css [documentation](https://github.com/h5bp/main.css/blob/master/README.md#features)
```

File: src/doc/TOC.md [hunk 3]
```diff
@@ -34,4 +34,4 @@ aspects of your website/web app (e.g.: the performance, security, etc.).
   * [Node.js](https://github.com/h5bp/server-configs-node)
 * [Front-end Developer Interview Questions](https://github.com/h5bp/Front-end-Developer-Interview-Questions)
 * [create-html5-boilerplate](https://github.com/h5bp/create-html5-boilerplate) — Quick start HTML5 Boilerplate development
-* [main.css](https://github.com/h5bp/main.css) — the main.css file included with HTML5 Boilerplate
+* [main.css](https://github.com/h5bp/main.css) — the main.css file included (as style.css) into HTML5 Boilerplate
```
Comment (roblarsen) [inline src/doc/TOC.md:37]:
```suggestion
* [main.css](https://github.com/h5bp/main.css) — the main.css file included (as style.css) with HTML5 Boilerplate
```

File: src/doc/css.md [hunk 4]
```diff
@@ -38,7 +38,7 @@ Several base styles are included that build upon `Normalize.css`. These styles:
 * and more...
 
 These styles are included in
-[main.css](https://github.com/h5bp/html5-boilerplate/blob/master/dist/css/main.css).
-See the [main.css](https://github.com/h5bp/main.css) project
-[documentation](https://github.com/h5bp/main.css/blob/master/README.md#features)
+[style.css](https://github.com/h5bp/html5-boilerplate/blob/master/dist/css/style.css)
+using [main.css](https://github.com/h5bp/main.css) project.
+See the [documentation](https://github.com/h5bp/main.css/blob/master/README.md#features)
```
Comment (roblarsen) [inline src/doc/css.md:43]:
```suggestion
See the main.css [documentation](https://github.com/h5bp/main.css/blob/master/README.md#features)
```

File: dist/doc/TOC.md [hunk 5]
```diff
@@ -34,4 +34,4 @@ aspects of your website/web app (e.g.: the performance, security, etc.).
   * [Node.js](https://github.com/h5bp/server-configs-node)
 * [Front-end Developer Interview Questions](https://github.com/h5bp/Front-end-Developer-Interview-Questions)
 * [create-html5-boilerplate](https://github.com/h5bp/create-html5-boilerplate) — Quick start HTML5 Boilerplate development
-* [main.css](https://github.com/h5bp/main.css) — the main.css file included with HTML5 Boilerplate
+* [main.css](https://github.com/h5bp/main.css) — the main.css file included (as style.css) into HTML5 Boilerplate
```
Comment (roblarsen) [inline dist/doc/TOC.md:37]:
```suggestion
* [main.css](https://github.com/h5bp/main.css) — the main.css file included (as style.css) with HTML5 Boilerplate
```

**Review by roblarsen — CHANGES_REQUESTED** [review]
Looks good! Please make the suggested changes. Thanks so much.

File: CHANGELOG.md (hunk 1 above)
Comment (sthiepaan) [reply CHANGELOG.md]: Actually I was relying with this change on what you did in #2341. I think in general this kind of thing should be done on release branch(?) ➡️ Reverted

**Review by roblarsen — APPROVED** [review]
👍

**Pull request merged**